index.js

```diff
@@ -0,0 +1,250 @@
+async function Connector(event) {
+
+    function serialize(request) {
+      const headers = {};
+      for (const entry of request.headers.entries()) {
+        headers[entry[0]] = entry[1];
+      }
+  
+      const serialized = {
+        url: request.url,
+        headers,
+        method: request.method,
+        mode: request.mode,
+        credentials: request.credentials,
+        cache: request.cache,
+        redirect: request.redirect,
+        referrer: request.referrer,
+      };
+      if (request.method !== "GET" && request.method !== "HEAD") {
```

Comment:
Is this really necessary? I know a GET is not supposed to put stuff in the body but I think that technically you can do it, and it'd be too bad to prevent users from using Ammo because we're being to strict